components/settings/index.js

```diff
@@ -157,15 +157,26 @@ function Settings (opts) {
 /**
  *
  */
-Settings.prototype.getDataUrl = function () {
-  // TODO pass in zoom level
+Settings.prototype.getDataUrl = function (zoom) {
+  console.info('getting data url for zoom level', zoom)
+  var fallback
   var backends = this.tabData.storage.backends
```

Comment:
Noticed that if I have two storages with ranges `[1, 15]` and `[16, 21]` and the zoom level is `15.1`, then I don't get a match so I just pick the first active storage as fallback to avoid not having a url at all.